src/main/java/javaslang/control/Applicative.java

```diff
@@ -0,0 +1,17 @@
+/*     / \____  _    _  ____   ______  / \ ____  __    _ _____
+ *    /  /    \/ \  / \/    \ /  /\__\/  //    \/  \  / /  _  \   Javaslang
+ *  _/  /  /\  \  \/  /  /\  \\__\\  \  //  /\  \ /\\/  \__/  /   Copyright 2014-now Daniel Dietrich
+ * /___/\_/  \_/\____/\_/  \_/\__\/__/___\_/  \_//  \__/_____/    Licensed under the Apache License, Version 2.0
+ */
+package javaslang.control;
+
+public interface Applicative<T> extends Apply<T> {
```

Comment:
This is a shortened version of my implementation of your code. The `@Override` method is the only one that satisfies the `Applicative` method.

```
public interface Validation<E,T> extends Kind<Validation<E,?>, T>, Applicative<Validation<E, ?>, T> {

    @Override
    default <B> Applicative<Validation<E, ?>, B> ap(Kind<Validation<E, ?>, ? extends Function<? super T, ? extends B>> f) {
        return null;
    }

    default <U> Validation<E, U> ap(Validation<E, ? extends Function<? super T, ? extends U>> v) {
        return null;
    }

    default <U> Validation<E, U> map(Function<? super T, ? extends U> f) {
        return null;
    }
    
}
```